lighthouse-core/gather/gather-runner.js

```diff
@@ -85,8 +85,9 @@ class GatherRunner {
     return Promise.resolve()
       // Begin tracing only if requested by config.
       .then(_ => options.config.recordTrace && driver.beginTrace(options.flags))
-      // Network is always recorded for internal use, even if not saved as artifact.
+      // Network is always recorded for gatherer use
       .then(_ => driver.beginNetworkCollect(options))
+      .then(_ => driver.beginDevtoolsLog())
```

Comment:
nit: should we do this the other way around? feasibly there could be some network request recorded before the devtools log start I guess, idk